nx.json

```diff
@@ -1,13 +1,20 @@
 {
   "extends": "nx/presets/npm.json",
-  "affected": { "defaultBase": "main" },
+  "affected": {
+    "defaultBase": "main"
+  },
   "parallel": 6,
+  "tui": {
+    "enabled": false
```

Comment:
why disable the tui? I thought it was much nicer to deal with than the current output